lib/ramble/ramble/workspace/workspace.py

```diff
@@ -1639,6 +1619,24 @@ def all_auxiliary_software_files(self):
         """Iterator over each file in $workspace/configs/auxiliary_software_files"""
         yield from self._auxiliary_software_files.items()
 
+    def workspace_paths(self):
+        """Dictionary of path replacements for workspace"""
+        if not hasattr(self, "_workspace_path_replacements"):
+            self._workspace_path_replacements = {
```

Comment:
Do we want to document these keywords?

Reply:
Thanks, I did mean to do that but forgot to.